wgengine/magicsock/magicsock.go

```diff
@@ -985,6 +1006,16 @@ func (c *Conn) determineEndpoints(ctx context.Context) ([]tailcfg.Endpoint, erro
 		return nil, err
 	}
 
+	portmap := make(chan netaddr.IPPort, 1)
+	go func() {
```

Comment:
Debugging is easier when goroutines are all named. Move this to a method on Conn and then `go c.getPortMapping` ?